Modelica/Resources/C-Sources/ModelicaStandardTables.c

```diff
@@ -865,9 +866,7 @@ void* ModelicaStandardTables_CombiTimeTable_init3(_In_z_ const char* fileName,
 
 void ModelicaStandardTables_CombiTimeTable_close(void* _tableID) {
     CombiTimeTable* tableID = (CombiTimeTable*)_tableID;
-    if (NULL == tableID) {
-        return;
-    }
+    assert(NULL != tableID);
```

Comment:
Just wondering if simply `assert(tableID)` could be the more idiomatic way of checking that a pointer in C  is not null?

Reply:
Also possible. I prefer it explicit (and not implicitly casted to boolean).

Reply:
I used to have the same preference, but many years I changed position because I felt I was working against the language design.  The top answer here covers my position rather well:
https://stackoverflow.com/questions/3825668/checking-for-null-pointer-in-c-c

Reply:
I also read there:

> Having a consistent style is sometimes more important than which style you pick.

This should be the case here. Let's consider it out of scope for this PR.

Reply:
That's right.  I didn't think of the numerous other non-null tests we already have.  Resolving.